scripts/reconciliation.py

```diff
@@ -0,0 +1,127 @@
+# -----------------
+# product reconciliation scripts
+# -----------------
+#
+# Smoke tests are designed to be run against live environments
+#
+# > python3 scripts/reconciliation.py {HOST}
+#
+# if no host is provided defaults to localhost:8000
+#
+# Note- require installation of corgi-bindings:
+
+import argparse
+import json
+import logging
+import sys
+
+import corgi_bindings
+import requests
+import urllib3
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+# setup logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.INFO)
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+# get args
+parser = argparse.ArgumentParser()
+parser.add_argument("host", nargs="?")
+args = parser.parse_args()
+
+logger.info("smoke-test: start")
+
+session = None
+if args.host is None:
+    session = corgi_bindings.new_session(corgi_server_uri="http://localhost:8000", verify_ssl=False)
+else:
+    session = corgi_bindings.new_session(corgi_server_uri=f"https://{args.host}", verify_ssl=False)
+
+# check specific streams [corgi ofuri, deptopia id]
+stream_corpus = [
+    # ["o:redhat:rhn_satellite:6.7", "93"],
+    # ["o:redhat:rhn_satellite:6.8", "50"],
+    # ["o:redhat:rhn_satellite:6.9", "72"],
+    # ["o:redhat:ansible_automation_platform:1.2", "2085"],
+    # ["o:redhat:ansible_automation_platform:2.0", "2030"],
+    # ["o:redhat:ansible_automation_platform:2.1", "2154"],
+    # ["o:redhat:ansible_automation_platform:2.2", "2340"],
+    ["o:redhat:openshift:4.9.z", "100"],
+    ["o:redhat:rhacm:2.4.z", "1444"],
+    ["o:redhat:rhel:8.6.0.z", "2302"],
```

Comment:
This is not the same stream we are ingesting in the UAT load document. In UAT we are loading 8.6.0 (not .z), that is current product id 58 in Deptopia. I think we should be able to accurately compare NVRs between this stream in Corgi, and Deptopia.

Reply:
I verified that the 8.6.0 stream was successful for all builds in Deptopia for the current data in corgi-stage.prodsec. I think we should adjust the test to compare full NVR, not just name and version in order to be more accurate.

Reply:
will fix type and adjust back to nvr